# Introdução à Ciência de Dados - UFPB
Professor: Yuri Malheiros

## Medidas de Centralidade e Dispersão

A partir do dataset `movies.csv` calcule a média, mediana e desvio padrão para as colunas:

### US Gross

In [11]:
# ADICIONE O SEU CÓDIGO AQUI
import pandas as pd
import numpy as np

movies_df = pd.read_csv("../icd/datasets/movies.csv")

print(movies_df['US Gross'].mean())
print(movies_df['US Gross'].median())
print(movies_df['US Gross'].std())

44002085.163744524
22019465.5
62555311.39066211


### Production Budget

In [12]:
# ADICIONE O SEU CÓDIGO AQUI

print(movies_df['Production Budget'].mean())
print(movies_df['Production Budget'].median())
print(movies_df['Production Budget'].std())

31069171.4484375
20000000.0
35585913.44464443


### Running Time

In [14]:
# ADICIONE O SEU CÓDIGO AQUI

print(movies_df['Running Time (min)'].mean())
print(movies_df['Running Time (min)'].median())
print(movies_df['Running Time (min)'].std())

110.19354838709677
107.0
20.171013677535413


### IMDB Rating

In [15]:
# ADICIONE O SEU CÓDIGO AQUI

print(movies_df['IMDB Rating'].mean())
print(movies_df['IMDB Rating'].median())
print(movies_df['IMDB Rating'].std())

6.283467202141901
6.4
1.2522899386004784


### IMDB Votes

In [16]:
# ADICIONE O SEU CÓDIGO AQUI

print(movies_df['IMDB Votes'].mean())
print(movies_df['IMDB Votes'].median())
print(movies_df['IMDB Votes'].std())

29908.64457831325
15106.0
44937.58233504963


### Análise dos Resultados

Quais colunas tiveram valores muito diferentes entre média e mediana? Quais os seus desvios padrão?

As colunas que tiveram valores muito diferentes entre media e mediana foram: Us Gross, Production Budget, IMDB Votes. Us Gross teve um desvio padrão de: 62555311.39066211; Production Budget teve um desvio padrão de: 35585913.44464443; IMDB Votes teve um desvio padrão de: 44937.58233504963.

Que conclusões podemos tirar desses resultados?

Podemos extrair que o custo de produção e os lucros dos filmes variam muito e são bem disperços. Assim como os votos do IMDB variam muito pelos filmes e não tem um centro de votos.